pkg/api/tlog.go

```diff
@@ -52,6 +53,39 @@ func GetLogInfoHandler(params tlog.GetLogInfoParams) middleware.Responder {
 		inactiveShards = append(inactiveShards, is)
 	}
 
+	stable := swag.BoolValue(params.Stable)
+	if stable {
+		ts := time.Now().Truncate(time.Duration(viper.GetUint("publish_frequency")) * time.Minute).UnixNano()
+		// key is treeID/timestamp, where timestamp is rounded down to the nearest X minutes
+		key := fmt.Sprintf("%d/%d", api.logRanges.ActiveTreeID(), ts)
```

Comment:
Need to fix this, this should be "give me the latest", not by key (because the job might not have run for that time window yet)